frontend-react/src/components/ManagePublicKey/ManagePublicKey.test.tsx

```diff
@@ -1,136 +1,280 @@
-import { screen, fireEvent } from "@testing-library/react";
+/* eslint-disable testing-library/no-unnecessary-act */
```

Comment:
Let's only disable when it's necessary rather than doing it at the top of the file:

```ts
/* eslint-disable testing-library/no-unnecessary-act */
act(() => { ... });
/* eslint-enable testing-library/no-unnecessary-act */
```

It's a little obnoxious to do this, but it's better for visibility into workarounds rather than disabling the rule wholesale.